lib/screens/orders/ordersScreen.dart

```diff
@@ -49,6 +51,27 @@ class _NewOrdersScreenState extends State<NewOrdersScreen> {
     super.dispose();
   }
 
+  // for pull refresh
+  RefreshController _refreshController =
+      RefreshController(initialRefresh: false);
+
+  void _onRefresh() async {
```

Comment:
Also this has been implemented in mainLanding / products page. Why re-write ? Can you not make a generic class that can be used anywhere instead of re-write ?

Reply:
yes